docs/app-payments/webhooks.md

```diff
@@ -216,4 +216,11 @@ It is possible to provide `paymentPointId` when creating or updating webhooks. T
 </td>
 </tr>
 </tbody>
-</table>
\ No newline at end of file
+</table>
+
+## Create webhook
+You can view and manage webhooks in the [MobilePay portal](https://portal.mobilepay.dk/developer/webhooks) or via [Webhooks Api](/api/wehooks)
+
+When creating an webhook you need to subscribe to events and app payments events are available only if you have App Payments product. The url must be `HTTPS` and unique per webhook. 
```

Comment:
When creating a webhook, you need to subscribe to events, and app payments events are available only if you have an App Payments product. The URL must be `HTTPS` and unique per webhook.

Reply:
Grammarly approves
![image](https://user-images.githubusercontent.com/87471390/190624233-97e38708-0aa2-433f-9427-96a3ce0070d0.png)

Reply:
fixed